### PR TITLE
User profile

### LINKED ISSUE #50: Add view/form for user profile

### PR DESCRIPTION
**Pull request by Dragemil**
Closes #50 
Screenshot_20201204_111607

### CODE REVIEW

File: mobile/lib/contracts/user_profile/contracts.dart [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import 'package:tree_of_a_kind/features/cqrs/cqrs_action.dart';
+
+part 'contracts.g.dart';
+
+@JsonSerializable()
+class UserProfileDTO {
+  final String id;
+  final String firstName;
+  final String lastName;
+  final DateTime birthDate;
```
Comment (BartoszChrostowski) [inline mobile/lib/contracts/user_profile/contracts.dart:11]: Idk whether Fluent has JustDate out of the box, but if it does please change it.

File: mobile/lib/features/login/view/login_form.dart [hunk 2]
```diff
@@ -93,7 +94,7 @@ class _LoginButton extends StatelessWidget {
       buildWhen: (previous, current) => previous.status != current.status,
       builder: (context, state) {
         return state.status.isSubmissionInProgress
-            ? const CircularProgressIndicator()
+            ? const LoadingIndicator()
```
Comment (BartoszChrostowski) [inline mobile/lib/features/login/view/login_form.dart:97]: 😍

File: mobile/test/features/home/view/home_page_test.dart [hunk 3]
```diff
@@ -50,42 +51,29 @@ void main() {
...
-          BlocProvider.value(
-            value: authenticationBloc,
-            child: MaterialApp(
-              home: HomePage(),
-            ),
-          ),
-        );
-        expect(find.text('test@gmail.com'), findsOneWidget);
-      });
+      // testWidgets('avatar widget', (tester) async {
```
Comment (BartoszChrostowski) [inline mobile/test/features/home/view/home_page_test.dart:54]: yagni?

**Review by BartoszChrostowski — APPROVED** [review]
+1

File: mobile/lib/contracts/user_profile/contracts.dart (hunk 1 above)
Comment (Dragemil) [reply mobile/lib/contracts/user_profile/contracts.dart:11]: No it doesn't, nor does c# at the moment.

**Pull request merged**